src/fakeData/fakeHealthcareProfessionals.ts

```diff
@@ -23,12 +23,20 @@ export function generateRandomCreateHealthcareProfessionalInputArray({ count = 5
 }
 
 function generateLocalizedNameInput(): gqlTypes.LocalizedNameInput {
-    return {
-        firstName: faker.person.firstName(),
-        middleName: faker.person.middleName(),
-        lastName: faker.person.lastName(),
-        locale: faker.helpers.enumValue(gqlTypes.Locale)
-    }
+    const randomLocal = faker.helpers.enumValue(gqlTypes.Locale)
+     return randomLocal == gqlTypes.Locale.EnUs
+            ? {
+                firstName: faker.person.firstName(),
+                middleName: faker.person.middleName(),
+                lastName: faker.person.lastName(),
+                locale: randomLocal
+            }
+            : {
+                firstName: fakerJA.person.firstName(),
+                middleName: 'ジョン',
```

Comment:
I had to hardcode the middle name since `fakerJA.person.middleName()` doesn't create Japanese middle names. Here is one of the outputs I got:
```
 [
  {
    firstName: '海翔',
    middleName: 'Emerson',
    lastName: '山本',
    locale: 'ja_JP'
  }
]
```